hal/src/main/native/sim/PortsInternal.h

```diff
@@ -32,7 +32,7 @@ constexpr int32_t kNumREVPDHChannels = 24;
 constexpr int32_t kNumPDSimModules = kNumREVPDHModules;
 constexpr int32_t kNumPDSimChannels = kNumREVPDHChannels;
 constexpr int32_t kNumDutyCycles = 6;
-constexpr int32_t kNumAddressableLEDs = 1;
+constexpr int32_t kNumAddressableLEDs = 6;
```

Comment:
There should be a similar change in `hal/src/main/native/systemcore/PortsInternal.h`